version.php

```diff
@@ -27,7 +27,7 @@
 $plugin->component = 'fileconverter_librelambda';
 $plugin->release = '2019063000';
 $plugin->version = 2019063000;
-$plugin->requires = 2018051700;
+$plugin->requires = 2017111309;
```

Comment:
I would expect travis to be testing against this branch

Reply:
https://github.com/catalyst/moodle-fileconverter_librelambda/blob/master/.travis.yml#L30